src/components/Autosuggest/Autosuggest.js

```diff
@@ -34,7 +34,7 @@ export class Autosuggest extends React.Component {
         this.handleWrapperKeyDown = memoize(this.handleWrapperKeyDown);
```

Comment:
I think this component returns all of the information it fetches from google api. I'm not sure how exactly it can define a postal code for a city in this case so maybe this is a reason it's missing.

Reply:
I think that using the `(regions)` place type you can search on a postal code but you don't get postal codes returned in the predictions. In other words, a limitation on Google's side. If you do need such additional metadata, perhaps see if you can make use of the [Place Details API](https://developers.google.com/places/web-service/details).

Reply:
I don't know how we can integrate `LocationSearch` component to Monster search if postcode is required in terms of story REACH-82. I think that point should be clarified with Ruud or Chris.
https://gis.stackexchange.com/questions/33966/google-geocoder-lookup-get-postal-code-by-country-and-city

Reply:
Postal code is indeed required for Monster. But I can also understand that it doesn't make much sense to return postal codes for (large) cities as they can span multiple postal codes. Although probably not very pretty or efficient, you could look into a solution like making two requests in sequence: first get the autocomplete predictions, then get additional details for the selected place id using the API I mentioned before (assuming that will include a postal code, which may depend on the place type you request autocomplete predictions for). This could perhaps also be done outside of the LocationSelector scope / on the application side.

This needs further investigation, maybe involve Chris. It would be good if we can determine asap. if this can be solved outside of OneUI or that it is indeed related to this component / PR.